judo.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/amagimedia/judo/v2/client"
 	judoMsg "github.com/amagimedia/judo/v2/message"
+	amagiPub "github.com/amagimedia/judo/v2/protocols/pub/amagipub"
```

Comment:
Can we move judo to v3 ? As this is release provides a new config interface we can move to v3